### PR TITLE
CSI-4747/ support hostDefinition ioGroup in operator

### PR DESCRIPTION
**Pull request by matancarmeli7**
Signed-off-by: matancarmeli7 <matan.carmeli7@gmail.com>

### CODE REVIEW

**Review by ELENAGER — APPROVED** [review]
LGTM

File: api/v1/hostdefinition_types.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ type Definition struct {
 	// +kubebuilder:validation:Optional
 	NodeNameOnStorage string `json:"nodeNameOnStorage"`
 	// +kubebuilder:validation:Optional
-	IOGroups []string `json:"ioGroups"`
+	IOGroups []int `json:"ioGroups"`
```
Comment (ArbelNathan) [inline api/v1/hostdefinition_types.go:48]: why it's not considered supported as a string?

Comment (matancarmeli7) [reply api/v1/hostdefinition_types.go:48]: because it contains `io_group's` ids, and they are integers

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**